src/dlgTriggerEditor.cpp

```diff
@@ -619,8 +619,42 @@ dlgTriggerEditor::dlgTriggerEditor(Host* pH)
                              .arg(hostName));
     toolBar2->setOrientation(Qt::Vertical);
 
-    QMainWindow::addToolBar(Qt::LeftToolBarArea, toolBar2);
+    // Inserting them in this order also causes the first one (the top toolbar)
+    // to be listed first in the QMainWindows's default context menu:
     QMainWindow::addToolBar(Qt::TopToolBarArea, toolBar);
+    QMainWindow::addToolBar(Qt::LeftToolBarArea, toolBar2);
+
+    // (Top) "Actions" toolbar:
+    mpAction_restoreEditorActionsToolbar = new QAction(tr("Restore Actions toolbar"), this);
+    // (Left) "Items" toolbar:
+    mpAction_restoreEditorItemsToolbar = new QAction(tr("Restore Items toolbar"), this);
```

Comment:
```suggestion
    // (Top) "Actions" toolbar:
    mpAction_restoreEditorActionsToolbar = new QAction(tr("Restore Actions toolbar", "This will restore that toolbar in the editor window, after a user has hidden them."), this);
    // (Left) "Items" toolbar:
    mpAction_restoreEditorItemsToolbar = new QAction(tr("Restore Items toolbar", "This will restore that toolbar in the editor window, after a user has hidden them."), this);
```

Reply:
Um, what about #5522? 😜

Reply:
Still legit, but I'd rather solve that issue in one fell swoop and not have piecework of different coding styles, don't you?

Reply:
This has been addressed, in the new manner that that issue 5522 suggests.